KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -0,0 +1,49 @@
+# [kubevirtci](README.md): Testing kubevirtci with locally provisioned cluster
```

Comment:
I think that adding this file is a bad idea: it has a big overlap with KUBEVIRT_LOCAL_TESTING (even the filename is almost indistinguishable). Any fix would have to be repeated twice and mismatch is bound to be created.

Reply:
What about having the info on this new file KUBEVIRTCI_LOCAL_TESTING.md (and renaming it),
and that the previous file KUBEVIRT_LOCAL_TESTING.md will reference to the kubevirtci one, and just add the additional steps ?

Reply:
I prefer to have one file, starting with pure kubevirtci testing, then a section with how to add kubevirt on top of it.